vlasovsolver/cpu_trans_pencils.cpp

```diff
@@ -1273,8 +1273,51 @@ void printPencilsFunc(const setOfPencils& pencils, const uint dimension, const i
       }
 
       ibeg  = iend;
+      ss << "\n";
+   }
+
+   for (const auto& [bin, binsPencils] : pencils.binsPencils) {
+      const auto& cells = pencils.binsCells.at(bin);
+      std::set<uint64_t> collisions;
+
+      ss << "Bin " << bin << " pencils: ";
+      if (binsPencils.empty()) {
+         ss << "EMPTY ";
+      }
+
+      for (auto pencil : binsPencils) {
+         ss << pencil << " ";
+      }
+
+      ss << "\n";
+
+      ss << "Bin " << bin << " cells: ";
+      if (cells.empty()) {
+         ss << "EMPTY ";
```

Comment:
Hopefully this won't ever trigger either.

Reply:
Ditto